src/services/database.service.js

```diff
@@ -84,19 +84,17 @@ export async function syncPlans() {
 
 		const priceString = price !== null ? price?.toString() ?? '' : ''
 
-		await _addVariant(
-			{
-				productId: variant.product_id.toString(),
-				variantId: variant.variantId,
-				variantEnabled: true,
-				name: variant.name,
-				description: variant.description,
-				price: parseInt(priceString),
-				billingCycle: null,
-				packageSize: packageSize,
-			},
-			variant.productName
-		)
+		await _addVariant({
+			productId: variant.product_id.toString(),
+			variantId: variant.variantId,
+			variantEnabled: true,
+			name: variant.name,
+			description: variant.description,
+			price: parseInt(priceString),
+			billingCycle: null,
+			packageSize: packageSize,
+			productName: variant.productName,
```

Comment:
du coup la fonction _addVariant na plus besoin de deuxieme parametre 
![image](https://github.com/For-Hives/ForVoyez/assets/28759336/c12aadcd-2644-4785-8ddd-326905929f73)